csv2bufr/resources/mappings/malawi_synop_bufr.json

```diff
@@ -1,6 +1,7 @@
 {
     "inputDelayedDescriptorReplicationFactor": [0,0],
     "unexpandedDescriptors": [301150,307080],
+    "masterTableVersionNumber":32,
```

Comment:
We now have `masterTableVersionNumber` in two places in this file (lines 4 and 13).  Can we consolidate somehow?

Reply:
changed for both this and unexpanded descriptors.